mon-aide-cyber-api/test/infrastructure/entrepots/postgres/EntrepotAideConcret.spec.ts

```diff
@@ -226,6 +233,42 @@ describe('Entrepot Aidé Concret', () => {
         etat: 'INEXISTANT',
       });
     });
+
+    it('Récupère l’Aidé ciblé même si le SIRET n’est pas persisté côté Brevo', async () => {
```

Comment:
`it('Indique un Siret "NON_DISPONIBLE" dans le cas d'un Aidé créé avant que le Siret soit persisté côté Brevo')`